app/controllers/people_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PeopleController < ApplicationController
+  def show
+    @person = Person.find(params[:id])
+
+    render :show_people
+  end
+
+  def show_tasks
+    this_person_id = params[:id]
+    @person = Person.find(this_person_id)
+    @tasks = Task.where(person_id: this_person_id)
```

Comment:
Since you've specified the `has_many` relationship in the Person model, you can just use `@person.tasks` rather than retrieving the tasks directly from the Task model.